views/timeline/simple.hbs

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <link title="timeline-styles" rel="stylesheet" href="../../static/css/timeline-simple.css">
+    <script src="../../static/js/simpleTimeline.js"></script>
+
+</head>
+
+<body>
+    <h1>Top stories over the last 24 hours</h1>
+    <div class="flex-parent">
+        <div class="input-flex-container">
+            {{#each data}}
+            {{#if firstArticle}}
+            <div class="input active">
+                <span data-year="{{title}}" data-info="{{formatedDate}}"></span>
+            </div>
+            {{else}}
+            <div class="input">
+                <span data-year="{{title}}" data-info="{{formatedDate}}"></span>
+            </div>
+            {{/if}}
+            {{/each}}
+        </div>
+        <div class="description-flex-container">
+            {{#each data}}
+            {{#if firstArticle}}
```

Comment:
If the two outputs of the if are so similar could it be simplified to:
`<p class="{{#if firstArticle}}active{{/if}}">{{title}}</p>`
or is this too messy?

Reply:
No that is a much better way of doing it! Will implement